content/tutorials/git_authentication/index.Rmd

```diff
@@ -0,0 +1,260 @@
+The purpose of this tutorial is to make clear recommendations about authentication for employees at INBO who use Git version control in their workflows. 
+
+If you do not have a GitHub account, you can [sign up for a new GitHub account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account).
+The installation of Git, R and RStudio at INBO, is done by IT administrators who follow [these installation instructions for admins](../../installation/administrator/).
+In addition, the useR is asked to do some extra steps after installation of [Git](../../installation/user/user_install_git/), [R](../../installation/user/user_install_r/) and [RStudio](../../installation/user/user_install_rstudio/).
+
+To authenticate means passing information (= credentials) that proves that you are exactly who you declare to be.
+Authentication is about securely accessing your GitHub account's resources either directly by yourself or on behalf of you by a third party.
+For instance, you can grant an R package the right to do stuff on your behalf (e.g. reading your e-mail address).
+You determine what can and what cannot be done and this can be different for various third parties.
+
+Most of the material in this tutorial is explained in length in https://happygitwithr.com/connect-intro.html.
+We will mainly follow [the advice given in the usethis package](https://usethis.r-lib.org/articles/articles/git-credentials.html).
+
+Authentication is a quite confusing and technical topic.
+In the [TL;DR section](#tldr) (Too Long; Didn't read) we just provide a summary of the recipe to follow without any explanation.
+Using this recipe will hopefully just work.
+If you want to get some intuition about what you are doing, you will need to read the remainder of the tutorial too (and also check out the weblinks included in this tutorial).
+
+## TL;DR
+
+First, enable two-factor authentication (2FA) for your GitHub account: follow [these steps](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app).
+You need a time-based one-time password app for 2FA to work.
+We recommend a mobile app which you can download on your smartphone such as `andOTP` (open source, Android) or `Google Authenticator` (closed source, Android or iOS).
+
+Next, in R, install the `usethis` package (this will also install packages `gert` and `gitcreds` which we use below):
+
+```{r}
+install.packages("usethis")
+```
+
+Use the following commands to configure some global git options (if you haven't done this already):
+
+```{r}
+?gert::git_config_global_set # read the help file
+gert::git_config_global_set("user.name", "Your Name") # change Your Name
+gert::git_config_global_set("user.email", "your@email.com") # change your@email.com
```

Comment:
Isn't that exactly the same as step 3 [here](https://inbo.github.io/tutorials/installation/user/user_install_git/#windows)?  In this case these steps would be superfluous for INBO users.

Reply:
Yes it is, but it won't hurt to keep these guidelines here. I will add a command to check if you have already set these global configuration values or not.

Reply:
See [291fc24](https://github.com/inbo/tutorials/pull/309/commits/291fc2421c49f14a4fc337203639ca91f8dfede3)

Reply:
Maybe also explain in some words that they don't need to set the name and email if they are set already?  Honestly, if I am directed to a manual with only code, I just copy, paste and run it without trying to understand it first.  (For curiosity I did this, and it indeed overwrote `ElsLommelen` with `Your Name`, so it won't hurt only if you read all code comments carefully. ;-) )  Admittedly, I don't know which is the best way to make sure users read and execute the comments behind the code, as it is not really attracting the attention (it seems part of the code).
(I'm just trying to avoid that people can misinterpret this to avoid helpdesk calls...)